### DISCUSSION TITLE
Hide menu on certain pages

### DISCUSSION THREAD

**Discussion by deltapapa01** [templaza/astroid-framework#230] [General]
Hello, is it possible to hide the menu on certain pages? Normally this is controlled by a module, which I don't need in the Astroid. I would like to hide the entire menu on the start page.

Many many thanks,
deltapapa

**Comment by Chacapamac** [comment]
I think an easy way will be by css.
Astroid is adding for each page/menu of your site “body” container a specific class by the name of your menu
- Menu “Home”  will be “body” class  —> “home-alias”
- Menu “Article 4  will be “body” class —> “article-4-alias”

Now you can hide elements by css on a specific page by targeting the specific element

As example on my test website —> http://joom4-astro.graf-test.ca/ 
if I want to hide only the menu on the home page, I have to add this css

`body.home-alias ul.astroid-nav.d-lg-flex {display: none !important;}`

**Note:** You can also add your own custom class to the “body” container by adding to the “Home” menu item a specific name.
In my example “myfrontpage” class will be added to the “body” container classes

![Screen Shot 2022-10-22 at 9 26 37 AM](https://user-images.githubusercontent.com/1907276/197343302-d0999e86-74bd-4280-b714-73a3d6ab2f89.jpg)

**Comment by deltapapa01** [comment]
Hi @Chacapamac,
thanks for the tip. I hadn't even thought in that direction. In the Helix Fw I did that with an extra template style, which I personally don't think is that greatest idea.
But with CSS it is much better solved in my opinion.

Mercy and have a nice weekend
delta papa

**Comment by Chacapamac** [comment]
Happy to help.
Nice weekend to You too!
